Makefile

```diff
@@ -267,6 +267,7 @@ prd/lib/: src/lib/d3-3.3.1.min.js \
 	cp -rf  $^ $@
 
 prd/lib/build.js: src/lib/jquery-2.0.3.min.js \
+	            src/lib/slip.js \
```

Comment:
We should use a compressed version of slip.js here. Look at the target `fastclick` in the `Makefile` to have an example how to do